nss-day-cohort-39/nutshell-veridian-vicuna
# Add Chat Friend

### Changes
- Chat list now renders "+" button next to each username
- Friend can be added by clicking the "+" button 

### Testing
- Click a "+" in the chat list to add a friend. 
- You should get an appropriate message depending on the friend status of the user.
- Try adding a new friend, try adding yourself, and try adding someone who is already a friend.
